src/eos-updater-apply.c

```diff
@@ -78,7 +78,6 @@ apply (GTask *task,
   GMainContext *task_context = g_main_context_new ();
   const gchar *update_id = eos_updater_get_update_id (updater);
   const gchar *update_refspec = eos_updater_get_update_refspec (updater);
-  const gchar *orig_refspec = eos_updater_get_original_refspec (updater);
```

Comment:
I don't really understand this commit - the code is not exporting the property just to propagate it, but it "stores" it locally on the `GDBusProxy` object not differently from how you store it in the updater data. The property is separately exported on the bus for convenience of other possible clients (but I don't believe it is in use now).